vulture/utils.py

```diff
@@ -10,6 +11,27 @@ class VultureInputException(Exception):
     pass
 
 
+def condition_is_unsatisfiable(condition):
+    """
+    Try to evaluate the given condition. Return true, if the evaluation
+    doesn't need the values of any variables and evaluates to False.
+    """
+    builtins = (vars(__builtins__) if hasattr(__builtins__, '__dict__') else
+                __builtins__)
```

Comment:
__builtins__ is actually an implementation detail (note the s at the end). Let's use the following at the top of the file (https://docs.python.org/2/library/__builtin__.html, https://docs.python.org/3/library/builtins.html):
```
try:
    # Python 3
    import builtins
except ImportError:
    # Python 2
    import __builtin__ as builtins
```

Then below we need to use `vars(builtins).items()`.